packages/calcite-components/src/components/combobox/combobox.tsx

```diff
@@ -1684,6 +1769,20 @@ export class Combobox
       <div ariaHidden="true" class={CSS.floatingUIContainer} ref={setFloatingEl}>
         <div class={classes} ref={setContainerEl}>
           <ul class={{ list: true, "list--hide": !open }}>
+            {this.selectAllEnabled &&
+              this.selectionMode !== "single" &&
+              this.selectionMode !== "single-persist" && (
+                <calcite-combobox-item
+                  ariaLabel="Select All"
+                  class={{ [CSS.selectAllCheckbox]: true }}
+                  id={`${this.guid}-select-all-enabled`}
+                  ref={this.setSelectAllComboboxItemReferenceEl}
+                  role="option"
+                  tabIndex="-1"
+                  text-label="Select All"
+                  value="Select All"
```

Comment:
Does this need a value? if so, can it be more of a value instead of a label?

Reply:
It looks like combobox-item warns if a `value` isn't provided.  Probably could have the value be `select-all` to make it more value-like.